test/utils.h

```diff
@@ -657,6 +647,14 @@ at::Tensor matmulAtInput2D(
     const int B = 0,
     const int device = 0);
 
+// Given a tensor view created by matmulAtInput2D or matmulAtInput3DTuring,
```

Comment:
A side question - will `matmulAtInput3DTuring` function be renamed to `matmulAtInput3D`, to follow the same route as other helper functions, when we retired Volta support?

Reply:
No, because there's also a `matmulAtInput3DHopperRS` and `matmulAtInput3DHopperSS` in `test_mma.cpp`.